<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>704</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1054</width>
    <height>704</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1054</width>
    <height>704</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home Amministratore</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>704</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>homeAmministratore.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestUtenti">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>GESTIONE UTENTI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestCorsi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>GESTIONE CORSI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestIngressi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>GESTIONE INGRESSI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestRicevute">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>GESTIONE RICEVUTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_logout">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>580</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestTurni">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>GESTIONE TURNI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestAvvisi">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>190</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>GESTIONE AVVISI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_gestAbbonamenti">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>340</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>GESTIONE ABBONAMENTI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_stats">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>480</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
background-color: white;
color: black;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>VISUALIZZA STATISTICHE</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Amminstratore.qrc"/>
 </resources>
 <connections/>
</ui>
